### ISSUE TITLE
[BUG] test_binance_market.py infinite loop

### ISSUE THREAD

**Issue by carlolm** [CoinAlpha/hummingbot#300]
**Describe the bug**
Running the binance test is infinite looping.
`python test/test_binance_market.py`

This repeats non-stop:

```
INFO:hummingbot.market.binance.binance_order_book_tracker:Diff messages processed: 86, rejected: 0, queued: 0
INFO:hummingbot.market.binance.binance_order_book_tracker:Processed 48 order book diffs for LOOMETH.
INFO:hummingbot.market.binance.binance_order_book_tracker:Processed 38 order book diffs for ZRXETH.
INFO:hummingbot.market.binance.binance_api_user_stream_data_source:Refreshed listen key 7EYROZaGbjKxHhSqmuy4yi8LB5Yt8mp9SrGYnEDokg65XAMMK8CXV67pnfEn.
INFO:hummingbot.market.binance.binance_order_book_tracker:Diff messages processed: 93, rejected: 0, queued: 0
INFO:hummingbot.market.binance.binance_order_book_tracker:Processed 50 order book diffs for ZRXETH.
INFO:hummingbot.market.binance.binance_order_book_tracker:Processed 43 order book diffs for LOOMETH.
```

**Steps To Reproduce**
1. Install and compile hummingbot from source
2. Run binance test

For detailed steps, see Jenkinsfile:
https://github.com/CoinAlpha/hummingbot/blob/dev/0.7.0/jenkins/tests/exchanges/binance/Jenkinsfile

...

**Screenshots**
// If applicable, add screenshots to help explain your problem.

**Release version**
dev/0.7.0 as of May 24, 2019

**Attachments**
[test-binance.log](https://github.com/CoinAlpha/hummingbot/files/3218949/test-binance.log)

**Optional: your discord username:**

**Comment by martinkou** [comment]
Some of the Binance market unit tests are not working correctly right now. e.g. #302.

I guess we should just leave it out for now, and create a task item to review and fix the Binance unit tests before we add it to CI.

**Comment by yzhang1994** [comment]
Looks like https://github.com/CoinAlpha/hummingbot/pull/302 was never merged due to 0.8.0 release. 
Was there another PR that solved the binance test issue?

**Comment by martinkou** [comment]
Should be fixed w/ #398.
